ads/custom.md

```diff
@@ -0,0 +1,172 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Custom
+
+Custom does not represent a specific network. Rather, it provides a way for 
+a site to display simple ads on a self-service basis. You must provide
+your own ad server to deliver the ads in json format as shown below.
+
+Each ad must contain a mustache template (see mustache.js).
+
+Each ad must contain the URL that will be used to fetch data from the server.
+
+Usually, there will be multiple ads on a page. The best way of dealing with this
+is to give all the ads the same ad server URL and give each ad a different slot id:
+this will result in a single call to the ad server.
+
+An alternative is to use a different URL for each ad, according to some format
+understood by the ad server.
+
+## Examples
+
+### Single ad
+
+```html
+<amp-ad width=300 height=250
+    type="custom"
+    data-slot="1"
+    data-url="https://mysite/my-ad-server"
```

Comment:
Done. I've used `?someparam=somevalue` to show that it can really be anything the implementor feels like.